server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/delivery/SieveMailet.java

```diff
@@ -20,14 +20,12 @@
 
```

Comment:
That's perhaps what I can improve for next refactoring.

However writing tests before code isolation is hard.